.gitattributes

```diff
@@ -8,3 +8,7 @@ phpstan.neon export-ignore
 phpunit.xml.dist export-ignore
 psalm.xml export-ignore
 rector.php export-ignore
+box.json.dist export-ignore
+composer-dependency-analyser.php export-ignore
+scoper.inc.php export-ignore
+docs export-ignore
```

Comment:
Isnt it useful to keep docs ?
Especially because there are listed in the readme

Reply:
I looked at Twig and they don't it. 

There might be a case where this is useful sometimes. But we should be conservative with what we archive and download every time on CI ;)

Reply:
Ok, can you update it too
```
.* export-ignore
patches export-ignore
docs export-ignore
tests export-ignore
infection.json export-ignore
Makefile export-ignore
box.json.dist export-ignore
composer-dependency-analyser.php export-ignore
phpstan.neon export-ignore
phpunit.xml.dist export-ignore
psalm.xml export-ignore
rector.php export-ignore
scoper.inc.php export-ignore
```
this way it's in the same order than in the repo, and

`composer-require-checker.json export-ignore` can be removed

Reply:
Done